flutter/lib/src/navigation/time_to_display_tracker.dart

```diff
@@ -0,0 +1,34 @@
+// ignore_for_file: invalid_use_of_internal_member
+
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+import '../../sentry_flutter.dart';
+import 'time_to_initial_display_tracker.dart';
+
+@internal
+class TimeToDisplayTracker {
```

Comment:
`l`: This class is a bit obsolete right now, as it is a wrapper around `TimeToInitialDisplayTracker`.

Reply:
TTFD is also going to be added to this class, not sure if I should refactor it now

Reply:
Then keep it. We can also ditch it later.